engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetStateParameter.java

```diff
@@ -28,46 +28,20 @@
  * See classes EnableBlending and EnableFaceCulling as working implementations.
  */
 abstract class SetStateParameter implements StateChange {
-    private int glParameter;
-    private boolean enabled;
+    protected int glParameter;
 
     /**
      * Construct an instance of this class, provided an OpenGL constant and a boolean to enable or disable
      * the indicated mode.
      *
      * @param glParameter An integer representing one of the many OpenGL constants, i.e. GL_DEPTH_TEST
-     * @param enabled A boolean indicating if the mode given by the parameter above must be enabled (true) or disabled (false).
      */
-    SetStateParameter(int glParameter, boolean enabled) {
+    SetStateParameter(int glParameter) {
         this.glParameter = glParameter;
-        this.enabled = enabled;
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(enabled, glParameter);
-    }
-
-    @Override
-    public boolean equals(Object obj) {
-        return (obj instanceof SetStateParameter) && this.enabled == ((SetStateParameter) obj).enabled && this.glParameter == ((SetStateParameter) obj).glParameter;
-    }
-
-    private String getStatus() {
-        return enabled ? "Enabled" : "Disabled";
-    }
-
-    @Override
-    public String toString() {
```

Comment:
getStatus was a public accessor not used anywhere, SetStateParameter still has a toString() (although much more generic that only prints class name, looks quite pretty in the taskList) and equals.. I don't know.

I'll take care of this one too.